Clients/src/presentation/components/Inputs/Dropdowns/index.tsx

```diff
@@ -0,0 +1,131 @@
+import { Grid, Typography, useTheme } from "@mui/material";
```

Comment:
@melpsh 
Thanks for the code 👍🏻🙏🏻
Now, here are some more comment by Copilot:

**This line seems fine. Just ensure that all the imported components are being used in the file. Unused imports should be removed to keep the code clean.**

So I guess it detects one of these as unused imports